packages/datagateway-dataview/public/res/default.json

```diff
@@ -211,7 +211,10 @@
   "buttons": {
     "add_to_cart": "Add to selection",
     "remove_from_cart": "Remove from selection",
-    "download": "Download"
+    "download": "Download",
+    "tooltip": {
+      "unable_to_download": "Unable to download - this file is empty"
```

Comment:
Additionally, this text is confusing when it's an investigation or dataset in question. How about replacing `file` with `item`?